packages/victory-core/src/victory-util/helpers.js

```diff
@@ -90,6 +90,13 @@ function getPadding(props) {
   };
 }
 
+function getBackgroundPadding(props) {
+  const { backgroundPadding } = props;
+  const padding = backgroundPadding;
+
+  return getPadding({ padding });
+}
+
```

Comment:
Instead of this, what about altering `getPadding` like so:

```
function getPadding(props, name = "padding") {
  const padding = props[name];
  const paddingVal = typeof padding === "number" ? padding : 0;
  const paddingObj = typeof padding === "object" ? padding : {};
  return {
    top: paddingObj.top || paddingVal,
    bottom: paddingObj.bottom || paddingVal,
    left: paddingObj.left || paddingVal,
    right: paddingObj.right || paddingVal
  };
}
```